dev_envs/mac-env-setup.md

```diff
@@ -0,0 +1,258 @@
+# Setting up a Mac-based development environment #
+
+## Initial Account Setup (as Administrator) ##
+
+For this section, log in as your Administrator account.
+
+### Account Permissions ###
+
+If you have a Standard user account, you’ll need to allow it access to run
+`sudo` commands. First, you’ll want to take note of your account name by
+viewing the folders listed inside of the `/Users/` folder - in this example,
+the account name is **username**.
+
+```bash
+$ ls /Users/
+username        Shared        fedadmin
+```
+
+Then, you’ll add your **username** to the `/etc/sudoers` file using `visudo`
+like this:
+
+```sh
+$ sudo visudo
+...
+##
+# User specification
+##
+
+# root and users in group wheel can run anything on any machine as any user
+root        ALL = (ALL) ALL
+%admin      ALL = (ALL) ALL
+username    ALL = (ALL) ALL
+```
+
+Press `Esc` followed by `:wq` to save and quit `visudo`.
+
+### Install XCode Command Line Tools ###
+
+Download XCode and Accept the License Agreement, then install the XCode
+Command Line Tools.
+
+```bash
+sudo xcodebuild -license
+xcode-select --install
+```
+
+### Docker ###
+
+Download the latest stable [Docker for Mac](https://www.docker.com/docker-mac).
+Install it by double-clicking the downloaded `dmg` file and dragging to the
+`Applications` folder.
+
+### iTerm2 (recommended) ###
+
+To set up iTerm2 - a MacOS terminal replacement with some nifty features, visit
+the [iTerm2 download page](https://www.iterm2.com/downloads.html) to get the
+latest stable release. Install it by double-clicking the downloaded `dmg` file
+and dragging to the `Applications` folder.
+
+Once installed, run the iTerm2 app and select the menu option
+`iTerm2 > Make iTerm2 Default Term`. NOTE: You may also need to set this
+setting as your Standard user account.
```

Comment:
Standard :arrow_right: standard

Reply:
Resolved in 802d2dd2215abd0eda70209fe993f0b5d8227ad4